llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -687,7 +689,10 @@ void LoadInst::dump() const {
 void StoreInst::setVolatile(bool V) {
   auto &Tracker = Ctx.getTracker();
   if (Tracker.isTracking())
-    Tracker.track(std::make_unique<SetVolatile>(this, Tracker));
+    Tracker.track(
```

Comment:
multi-line ifs should have braces

Reply:
I fixed the ones related to this patch, but there are plenty more. I will fix them separately.